doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -102,3 +106,14 @@ You can open the Cockpit console for a virtual machine from the image:ssui-acces
 image:cockpit-login-nb.png[Cockpit Login]
 . Once you have logged in, you will see the tabs for the dashboard and the individual machines added to Cockpit. 
 image:cockpit-ui-nb.png[Cockpit UI]
+
+[[vm-power-operations]]
+=== Power Operating the Virtual Machine
+
+Follow this procedure to perform power operations on virtual machines, that is start, stop, or suspend, depending on the current state of the virtual machine. 
+
+. From the *Services* screen, click on the required service.
+. Under *Resources*, click more options _(represented by three dots)_ where you have the option to start, stop, or suspend the virtual machine.
+. Alternatively, click on the virtual machine name to see more details about the resource, including the *Power Operations* button. 
```

Comment:
Should this sentence (Alternatively, click on..." be part of step 2, since it's an alternative step and not continuing on in the sequence?

Reply:
Fixed.